{{cookiecutter.repo_name}}/development.ini

```diff
@@ -24,6 +24,10 @@ sqlalchemy.url = sqlite:///%(here)s/{{ cookiecutter.repo_name }}.sqlite
 # wsgi server configuration
 ###
 
+[alembic]
+# path to migration scripts
+script_location = {{ cookiecutter.repo_name }}/alembic
+
```

Comment:
I recommend adding:

```
file_template = %%(year)d_%%(month).2d_%%(day).2d_%%(hour).2d_%%(minute).2d_%%(second).2d_%%(rev)s
```

Or maybe a bit shorter version there, to have sortable revisions. The default hash value makes it impossible to find out what is the newest revision in a date-less system, like Git.